clang/test/CIR/CodeGen/call-extra-attrs.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -O2 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
```

Comment:
You also need to add RUN lines that check for the final LLVM output, see `clang/test/CIR/CodeGen/abstract-cond.c` for a RUN line example.